client/src/styles/components/_chats-list.scss

```diff
@@ -1,103 +1,148 @@
-.chats-list {
-  display: flex;
-  flex-direction: column;
-  padding: 0 2rem;
+.chats-list-page {
   height: 100vh;
+  width: 100vw;
+  position: fixed;
 
-  .header {
-    margin-top: 2.625rem;
-    color: #0e1c74;
-    font-size: 1.5rem;
-    font-weight: 500;
-    margin-bottom: 1.75rem;
-    font-family: $font-secondary;
-  }
-
-  .list {
+  &>.content {
+    position: fixed;
+    width: 100vw;
+    height: 65vh;
```

Comment:
This is a very bad practice that I've seen in the code before but didn't know who wrote it. You should almost never use specific vh/vw units for 2 main reasons:

1. It's not necessarily accurate to the figma design
2. When the screen is changing it might not always scale how we'd expect. For example - earlier in the project someone made the send icon be relative to the screen width. As you can see in the figma design it's always a circle. So the actual result was that it shrinked/widened incorrectly when it shouldn't even change size

Try to almost always use flexbox, margins and paddings to make it accurate to the design in figma. If you didn't know, you can press the toggle on the top right to switch to 'developer mode'. This allows you to click on any element in figma to view the paddings and margins in pixels. Saves a lot of time, calculations and effort

Reply:
Interesting, I actually didn't know about the developer mode, though from what I heard using px is not a good practice and rem is preferable.

Reply:
Rem and px is controversial most of the time. font sizes 100% needs to be with rem, it's the margins and paddings that are not always the same. I personally always use rem just because I like it. I still think it needs to be changed to respond to the layout rather than "fixating on itself" with margins and paddings in addition to the first reason. It's just hard to predict where it would fail basically